kitsune/wiki/views.py

```diff
@@ -520,6 +520,12 @@ def review_revision(request, document_slug, revision_id):
     revision_contributors = list(set(
         based_on_revs.values_list('creator__username', flat=True)))
 
+    # Get latest revision which is still not approved. Use "latest_revision.id" to get the id.
+    latest_revision = Revision.objects.filter(document=doc, is_approved=False).order_by
+    ('-created')[0]
```

Comment:
It doesn't work because this line is split incorrectly. Python reads these two lines as two entirely separate expressions. Because of this `latest_revision` is equal to the `order_by` function, which is uncalled. You need to combine these on the same line, or if that is too long, put parenthesis around the entire expression.

In general if you have to wrap lines, don't do it write before a parenthesis like this. Do it after a set of parenthesis, or before a `.`.